news/_posts/core-weekly/2020-06-02-coreweekly-22-2020.md

```diff
@@ -0,0 +1,248 @@
+* [#1](https://github.com/PrestaShop/LocalizationFiles/pull/1): Port updates from develop branch (as of 2020-05-13), by [@eternoendless](https://github.com/eternoendless)
+
+
+### PrestaShop Virtual Machine
+* [#6](https://github.com/PrestaShop/vagrant/pull/6): Add fixtures, by [@SimonGrn](https://github.com/SimonGrn)
+
+
+### Translation Files repository
+* [#10](https://github.com/PrestaShop/TranslationFiles/pull/10): Add Kurdish, by [@Progi1984](https://github.com/Progi1984)
+* [#9](https://github.com/PrestaShop/TranslationFiles/pull/9): Add translations files for 1.7.8, by [@Progi1984](https://github.com/Progi1984)
+
+
+### Changes in developer documentation
+* [#544](https://github.com/PrestaShop/docs/pull/544): Product form documentation, by [@jolelievre](https://github.com/jolelievre)
+* [#543](https://github.com/PrestaShop/docs/pull/543): Fix minor mistakes in contribution guidelines, by [@eternoendless](https://github.com/eternoendless)
+* [#542](https://github.com/PrestaShop/docs/pull/542): Add notice warning for webservice usage, by [@matks](https://github.com/matks)
+
+
+### TranslationTools Bundle
+* [#77](https://github.com/PrestaShop/TranslationToolsBundle/pull/77): Add CONTRIBUTORS file, fix composer.json, by [@matks](https://github.com/matks)
+
+
+### The PrestaShop open source project
+* [#38](https://github.com/PrestaShop/open-source/pull/38): Adding #community-help slack channel, by [@ttoine](https://github.com/ttoine)
+
+
+### PrestaShop Specifications
+* [#121](https://github.com/PrestaShop/prestashop-specs/pull/121): Clean an reorganize the repo. Thank you [@sam-pires](https://github.com/sam-pires)
+* [#119](https://github.com/PrestaShop/prestashop-specs/pull/119): Create multistore specs for ps_linklist, by [@marionf](https://github.com/marionf)
+* [#101](https://github.com/PrestaShop/prestashop-specs/pull/101): Create Create new order.md. Thank you [@MatShir](https://github.com/MatShir)
+* [#95](https://github.com/PrestaShop/prestashop-specs/pull/95): Submit improvements, by [@LouiseBonnard](https://github.com/LouiseBonnard)
+* [#87](https://github.com/PrestaShop/prestashop-specs/pull/87): Create catalog-products-listing.md, by [@marionf](https://github.com/marionf)
+
+
+### Registered customer information
+* [#6](https://github.com/PrestaShop/statspersonalinfos/pull/6): Release 2.0.3, by [@Progi1984](https://github.com/Progi1984)
+* [#5](https://github.com/PrestaShop/statspersonalinfos/pull/5): Fix age data with birthday 0000-00-00 and empty currency label, by [@Matt75](https://github.com/Matt75)
+
+
+### Gamification module
+* [#73](https://github.com/PrestaShop/gamification/pull/73): Rename function name to avoid conflicts, by [@Quetzacoalt91](https://github.com/Quetzacoalt91)
+
+
+### Traces
+* [#9](https://github.com/PrestaShop/traces/pull/9): Changed arguments for username & password in options, by [@Progi1984](https://github.com/Progi1984)
+
+
+### example_module_mailtheme
```

Comment:
No "pretty" name

Reply:
Nothing to do here, this is the title in the repos's readme.md